### PR TITLE
Using edX Blue for DataTables Sort Icons

### PR DESCRIPTION
**Pull request by clintonb**
- Updated DataTables to use FontAwesome icons in place of images
- Changed color of active sort icon to edX blue

@dsjen @rocha

### CODE REVIEW

**Comment by dsjen** [comment]
Do you have a screenshot of the new blue in action?

**Comment by clintonb** [comment]
No. It's just a barely noticeable color change (despite the large amount of
CSS).

## 

Clinton Blackburn

**Comment by dsjen** [comment]
I'd appreciate a screenshot please.

**Comment by clintonb** [comment]
@dsjen 

screen shot 2014-09-05 at 9 34 53 am

screen shot 2014-09-05 at 9 35 14 am

File: analytics_dashboard/static/vendor/dataTables/dataTables.fontAwesome.scss [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+
+table.dataTable thead th.sorting:after,
+table.dataTable thead th.sorting_asc:after,
+table.dataTable thead th.sorting_desc:after {
+	position: absolute;
+	top: 12px;
+	right: 8px;
+	display: block;
+	font-family: FontAwesome;
+}
```
Comment (marcotuts) [inline analytics_dashboard/static/vendor/dataTables/dataTables.fontAwesome.scss:22]: Only suggestion on this PR is to add a bit more  (5px or so) separation between the text label area and the font awesome sort icons.

Comment (clintonb) [inline analytics_dashboard/static/vendor/dataTables/dataTables.fontAwesome.scss:22]: Done.
screen shot 2014-09-05 at 12 57 41 pm

Comment (marcotuts) [inline analytics_dashboard/static/vendor/dataTables/dataTables.fontAwesome.scss:22]: sweet! :thumbsup:  from my perspective.

**Comment by dsjen** [comment]
Thank you for sharing the screenshot.  I never got a chance to review this after my initial comments and would like to in the future.

**Pull request merged**